day_two_guide/topics/proc_restoring-etcd.adoc

```diff
@@ -11,253 +11,37 @@ Module included in the following assemblies:
-+
-----
-# rm -Rf /var/lib/etcd/*
-----
-+
-[NOTE]
-====
-In an all-in-one cluster, the etcd data directory is located in the
-`/var/lib/origin/openshift.local.etcd` directory.
-====
-
-. Restore a healthy backup data file to each of the etcd nodes. Perform this step on all etcd hosts, including master hosts collocated with
-etcd.
-+
-----
-# cp -R /backup/etcd-xxx/* /var/lib/etcd/
-# mv /var/lib/etcd/db /var/lib/etcd/member/snap/db
-# chcon -R --reference /backup/etcd-xxx/* /var/lib/etcd/
-----
-
-. Run the etcd service on one of your etcd hosts, forcing a new cluster.
-+
-This creates a custom file for the etcd service, which overwrites the execution
-command adding the `--force-new-cluster` option:
-+
-----
-# mkdir -p /etc/systemd/system/etcd.service.d/
-# echo "[Service]" > /etc/systemd/system/etcd.service.d/temp.conf
-# echo "ExecStart=" >> /etc/systemd/system/etcd.service.d/temp.conf
-# sed -n '/ExecStart/s/"$/ --force-new-cluster"/p' \
-    /usr/lib/systemd/system/etcd.service \
-    >> /etc/systemd/system/etcd.service.d/temp.conf
-
-# systemctl daemon-reload
-# master-restart etcd
-----
-
-. Check for error messages:
-+
-----
-# master-logs etcd etcd
-----
-
-. Check for health status:
-+
-----
-# etcdctl3 cluster-health
-member 5ee217d17301 is healthy: got healthy result from https://192.168.55.8:2379
-cluster is healthy
-----
-
-. Restart the etcd service in cluster mode:
-+
-----
-# rm -f /etc/systemd/system/etcd.service.d/temp.conf
-# systemctl daemon-reload
-# master-restart etcd
-----
-
-. Check for health status and member list:
-+
-----
-# etcdctl3 cluster-health
-member 5ee217d17301 is healthy: got healthy result from https://192.168.55.8:2379
-cluster is healthy
-
-# etcdctl3 member list
-5ee217d17301: name=master-0.example.com peerURLs=http://localhost:2380 clientURLs=https://192.168.55.8:2379 isLeader=true
-----
-
-. After the first instance is running, you can add the remaining peers back into the cluster.
-
-=== Fix the `peerURLS` parameter
-
-After restoring the data and creating a new cluster, the `peerURLs` parameter
-shows `localhost` instead of the IP where etcd is listening for peer
-communication:
-
-----
-# etcdctl3 member list
-5ee217d17301: name=master-0.example.com peerURLs=http://*localhost*:2380 clientURLs=https://192.168.55.8:2379 isLeader=true
-----
-
-==== Procedure
-
-. Get the member ID using `etcdctl member list`:
-+
-----
-`etcdctl member list`
-----
-
-. Get the IP where etcd listens for peer communication:
-+
-----
-$ ss -l4n | grep 2380
-----
-
-. Update the member information with that IP:
-+
-----
-# etcdctl3 member update 5ee217d17301 https://192.168.55.8:2380
-Updated member with ID 5ee217d17301 in cluster
-----
-
-. To verify, check that the IP is in the member list:
-+
-----
-$ etcdctl3 member list
-5ee217d17301: name=master-0.example.com peerURLs=https://*192.168.55.8*:2380 clientURLs=https://192.168.55.8:2379 isLeader=true
-----
-endif::[]
-
-== Restoring etcd snapshot
```

Comment:
@geliu2016 / @hexfusion 

Removed this procedure and kept only the next procedure for restoring when etcd is a static pod. Let me know if there is a reason that we would need to keep this procedure for of etcd is not a static pod.